README.md

```diff
@@ -136,12 +137,30 @@ Or you can import the CSS:
 
 Here's how you can get started developing locally:
 
+1. Clone this repo and create a symlink:
+
     $ git clone https://github.com/amarofashion/react-credit-cards.git
     $ cd react-credit-cards
     $ npm install
+    $ npm link
+
+2. Download the demo source from [codesandbox](https://codesandbox.io/s/ovvwzkzry9) and install the dependencies:
+
```

Comment:
Maybe we should add the unzip step just to be extra clear?

Reply:
Sure, doesn't hurt to be extra clear here :)